src/index.html

```diff
@@ -42,16 +42,23 @@
       </nav>
 
 <!--      landing view  -->
-      <main id="landing-view" class="landing-view">
+      <main id="landing-view" class="landing-view ">
         <section class="search-section">
           <!--          <form>-->
           <div class="search">
-            <label for="searchBar"></label>
-            <input type="search" class="search-bar" id="searchBar" name="search-bar" placeholder="Search by recipe name or ingredient">
-            <label for="searchButton"></label>
-            <button class="search-button" id="searchButton" name="search-button">Search</button>
+            <label for="searchBarLanding"></label>
+            <input type="text"
+                   class="search-bar"
+                   id="searchBarLanding"
```

Comment:
Looks like you've done some nice formatting on the search bar! Looks like the search bar will come first, and then the dropdown? I've done some work on the dropdown (just the text not the formatting) so we'll need to be careful with the merge here.